addons/sale/__manifest__.py

```diff
@@ -79,13 +79,12 @@
         'web.qunit_suite_tests': [
             'sale/static/tests/product_configurator_tests.js',
             'sale/static/tests/sales_team_dashboard_tests.js',
-            'sale/static/tests/sale_progressbar_field_tests.js'
         ],
         'web.report_assets_common': [
             'sale/static/src/scss/sale_report.scss',
         ],
         'web.assets_qweb': [
-            'sale/static/src/xml/sales_team_progress_bar_template.xml',
+            'sale/static/src/xml/**/*',
```

Comment:
@Arcasias this is the only place I would live unchanged, Sales app prefers explicit declarations.

Reply:
Finally I will be putting several files here after the whole owl migration. Your approach is better :)